### DISCUSSION TITLE
Is it possible to have link to multiple identities of same external Idp?

### DISCUSSION THREAD

**Discussion by Lan-Phan** [zitadel/zitadel#5740] [Q&A] [answered]
Hello,

I know that it's easy to add multiple external Idps to our system, but I concern about another thing: for 1 external Idp, is it possible for us to link to multiple identities? For ex., user ABC in Zitadel can be linked to:
+ account abc@gmail.com of Google Idp
+ account def@gmail.com of Google Idp

**Comment by hifabienne** [comment, marked as answer]
Hei @Lan-Phan
This should be possible, as long as you don't do it the other way around and link one Google IDP to multiple ZITADEL accounts.

**Comment by Lan-Phan** [comment]
Hi @hifabienne 

Seems you misunderstood my question because your example is about 1 google account linked to multiple Zitadel accounts.
I'd like to define more clearly my case:
* 1 org - 1 instance Zitadel
* Setup to have 1 external Google IdP
* is it possible to have account A in Zitadel linking to account abc@gmail.com + def@gmail.com from Google IdP?

Thanks.

**Reply by hifabienne** [reply]
Hei I understood you correctly. 
And I told that this is possible.
I just wanted to state that the other way would not be possible. but your case is.

**Comment by lanphan** [comment]
Hi @hifabienne 

Great, thanks so much.
